common/djangoapps/enrollment/management/tests/test_enroll_user_in_course.py

```diff
@@ -0,0 +1,87 @@
+""" Test the change_enrollment command line script."""
+
+import ddt
+import unittest
+
+from django.core.cache import cache
+from django.test import TestCase
+from django.test.utils import override_settings
+from django.conf import settings
+
+from django.core.management import call_command
+
+from enrollment.tests import fake_data_api
+from enrollment.api import get_enrollment
+from student.tests.factories import UserFactory
+
+
+@ddt.ddt
+@override_settings(ENROLLMENT_DATA_API="enrollment.tests.fake_data_api")
+@unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Test only valid in lms')
+class EnrollManagementCommandTest(TestCase):
+    """
+    Test the enroll_user_in_course management command
+    """
+
+    USER_EMAIL = "ralph@example.com"
+    COURSE_ID = "some/great/course"
+
+    def setUp(self):
+        super(EnrollManagementCommandTest, self).setUp()
+        fake_data_api.reset()
+        UserFactory(username='ralph_user', email=self.USER_EMAIL)
+        cache.clear()
+
+    def test_enroll_user(self):
+
+        fake_data_api.add_course(self.COURSE_ID, course_modes=['audit'])
+        command_args = {
+            'course': self.COURSE_ID,
+            'email': self.USER_EMAIL,
+        }
+
+        call_command(
+            'enroll_user_in_course',
+            **command_args
+        )
+
+        user_enroll = get_enrollment('ralph_user', self.COURSE_ID)
+        self.assertTrue(user_enroll['is_active'])
+
+    def test_enroll_user_twice(self):
+
+        fake_data_api.add_course(self.COURSE_ID, course_modes=['audit'])
+        command_args = {
+            'course': self.COURSE_ID,
+            'email': self.USER_EMAIL,
+        }
+
+        call_command(
```

Comment:
To make it even DRYer, consider `ddt`ing the previous test with an optional parameter for how many times to call the enroll management command.